components/brave_rewards/resources/extension/brave_rewards/components/panel.tsx

```diff
@@ -242,6 +246,9 @@ export class Panel extends React.Component<Props, State> {
       case 'backupWallet':
         clickEvent = this.onBackupWallet.bind(this, id)
         break
+      case 'ads-launch':
```

Comment:
should this not follow Lower Camel Case like `backupWallet`?

Reply:
@tmancey I agree that it should but I would have to open up a UI PR at this point to update it here, I think I can fix this at a later time

Reply:
Can you please raise a ticket for this refactor work, thanks